src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = days * 40;
+  const priceLong = days * 40 - 50;
+  const priceShort = days * 40 - 20;
```

Comment:
This calculations should be done only when conditions are met
Not need to calculate them beforehand
```suggestion
  const priceLong = price - 50;
  const priceShort = price - 20;
```